OPHD/States/MapViewStateHelper.cpp

```diff
@@ -578,8 +578,8 @@ NAS2D::Xml::XmlElement* writeResearch(const ResearchTracker& tracker)
 			"current",
 			{{
 				{"tech_id", techId},
-				{"progress", std::get<0>(values)},
-				{"assigned", std::get<1>(values)},
+				{"progress", values.progress},
```

Comment:
You know what... I see your point here. It's been what, a month since I wrote this code? And looking at it here in this commit after being off my feet for two weeks with COVID-19 makes it so, so, so easy to understand what this is supposed to mean.